docs/self-hosted/kubernetes-embedded.md

```diff
@@ -37,6 +37,11 @@ in an existing Kubernetes cluster using Helm follow [this guide](./kubernetes-he
   * s3.kurl.sh (required to install the kots CLI)
   * amazonaws.com (required to install the kots CLI)
 
+> Please note that a multi-node setup with kURL is only possible if no built-in services (postgres, minio, redis) are used.
```

Comment:
Just suggesting/discussing:

- I would move this after the first paragraph in the `Configure Private Packagist Self-Hosted` section
- Not sure if we need to, but maybe stress the reason why this is not a good option too so there can be no doubt about it (ie. local file storage):
 
 ```suggestion
> Please note that the built-in services (PostgreSQL, MinIO, Redis) are meant to be used with a single-node setup only, because these options rely on local file storage.
```

Reply:
I think your suggestion makes sense 👍

Reply:
I also moved the notice under the `Configure Private Packagist Self-Hosted` section.